### PR TITLE
PR for logout bug. 

### LINKED ISSUE #22: Logout Bug(?)
When logging out, then logging back in, it auto-logs in without giving me the opportunity to either use a different email or sign in using a different strategy. This makes me wonder if the logout functionality is actually working correctly

### PR DESCRIPTION
**Pull request by jduong09**
Resolves #22 

Log out feature was not implemented correctly.
Fixed feature by making request to correct auth0 endpoint.

### CODE REVIEW

**Comment by jduong09** [comment]
Taking a look at what occurs when a user logins with incorrect information.

File: auth.js [hunk 1]
```diff
@@ -10,22 +10,24 @@ require('dotenv').config();
...
 * My understanding of passport API:
+* Frontend: User sees log in button. They click log in button.
+* Frontend: Browser sends a request to http://localhost:5000/auth/login from the <a> tag.
+* Backend: Request hits express route /auth/login, where passport api logins user with the auto0 strategy.
 * 
 */
 
 router.get(
-  '/login',
+  '/login',
```
Comment (ziggysauce) [inline auth.js:20]: get rid of trailing space

```suggestion
  '/login',
```

File: auth.js [hunk 2]
```diff
@@ -10,22 +10,24 @@ require('dotenv').config();
...
+* Backend: Request hits express route /auth/login, where passport api logins user with the auto0 strategy.
 * 
 */
 
 router.get(
-  '/login',
+  '/login', 
   passport.authenticate('auth0', {
-    scope: "openid"
+    scope: "openid",
```
Comment (ziggysauce) [inline auth.js:22]: use single quotes here

```suggestion
    scope: 'openid',
```

File: auth.js [hunk 3]
```diff
@@ -46,29 +48,24 @@ router.get('/callback', (req, res, next) => {
       if (error) {
         return next(error);
       }
-
+
```
Comment (ziggysauce) [inline auth.js:51]: get rid of spaces

```suggestion

```

**Review by ziggysauce — COMMENTED** [review]
Nice, this is working when tested! Just some minor syntax changes, and then you can merge this in!

**Pull request merged**